.coveragerc

```diff
@@ -1,3 +1,9 @@
 # .coveragerc to control coverage.py
 [run]
 branch = True
+include =
+    aiohttp_admin/**/*.py
+    tests/**/*.py
+    admin-js/src/**/*.js
```

Comment:
@Dreamsorcerer coveragepy doesn't know anything about js, unless you add some pluging that would surface this info. So marking js files for coverage collection doesn't make sense.

Reply:
That's why I gave up on this approach. As you've seen, I've added examples/ to the integration test, and jest should report coverage for all JS files (except, it doesn't report coverage for test files yet, I've opened an issue for that).